### DISCUSSION TITLE
`junit-dep-4.11.20120805.1225.jar` cannot be found during intranet compilation

### DISCUSSION THREAD

**Discussion by zenvzenv** [open-telemetry/opentelemetry-java-instrumentation#9623] [Q&A]
When I imported the project into idea on the intranet, I got the following error:

```text
* What went wrong:
Could not find `junit-dep-4.11.20120805.1225.jar` (junit:junit-dep:4.11.20120805.1225).
Search in the following locations:
   xxx/junit/junit-dep/4.11.20120805.1225/junit-dep-4.11.20120805.1225.jar
```
I found the `junit-dep-4.11.20120805.1225` coordinates in `https://mvnrepository.com/`, but there are only pom files and no jar packages. How can I circumvent this problem by importing the project?

**Comment by laurit** [comment]
That pom contains a relocation so it not having a jar is expected.
